project-base/config/packages/prod/routing.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+    router:
+        strict_requirements: null
```

Comment:
Separation of files is recommended in Flex because each file is created on-demand with a different recipe. 
I know we already have these bundles installed and configured, so it doesn't matter, but I think keeping file structure as similar as possible with Symfony's default is better for adoption SSFW.